client/web/src/featureFlags/featureFlags.ts

```diff
@@ -18,6 +18,7 @@ export type FeatureFlagName =
     | 'search-results-keyboard-navigation'
     | 'enable-streaming-git-blame'
     | 'plg-enable-add-codehost-widget'
+    | 'sourcegraph-operator-only-observability'
```

Comment:
Looking a bit closer, should we extend this flag to cover all things we want to hide under the `Maintainence` category? Other pages we can hide:

- Updates (customers have no control over this)
- Migrations (customers also have no control over this)

Then the flag might be:

```suggestion
    | 'sourcegraph-operator-site-admin-hide-maintainence'
```